batavia/modules/base64.js

```diff
@@ -0,0 +1,61 @@
+batavia.modules.base64 = {
+	__doc__: "",
+	__name__: "base64",
+	__file__: "base64.js",
+	__package__: "",
+
+	b64encode: function(data){
+		var foo = batavia.types.Str(data).slice(2, -1)
+		return "b'" + new batavia.types.Str(window.btoa(foo)) + "'";
```

Comment:
While this prints out as the same thing as CPython, it isn't returning a `bytes` object.